### DISCUSSION TITLE
How to debug?

### DISCUSSION THREAD

**Discussion by batonac** [woocommerce/woocommerce-gutenberg-products-block#11789] [Q&A - Extension Integrations and Customizations]
Hey ~ First off, I'm thrilled to see all the modernization efforts happening in this initiative. Keep up the good work.

I'm experiencing an issue with the Checkout block where the cart items aren't rendering in the order summary:
![image](https://github.com/woocommerce/woocommerce-blocks/assets/4996285/5217227d-1663-4102-b58a-9f6b7da4688b)

This issue is quite perplexing, especially considering that the API is certainly returning the cart items correctly:
![image](https://github.com/woocommerce/woocommerce-blocks/assets/4996285/9003e398-ab87-4246-9998-07262a292b5d)

The issue first surfaced with the block in the WooCommerce plugin, and can be replicated in this feature plugin. The classic shortcode-based checkout still works.

I'm not seeing any errors in the WordPress debug.log or the browser console log that would seem to relate to this... how can I facilitate a verbose logging from these blocks in order to analyze what's happening here?

**Comment by opr** [comment]
Hi @batonac!

> Hey ~ First off, I'm thrilled to see all the modernization efforts happening in this initiative. Keep up the good work.

Thank you for your compliment!

> This issue is quite perplexing, especially considering that the API is certainly returning the cart items correctly:

Yes, it is, I see them in the data store too, so I know they're being moved from the API response into `wc/store/cart` correctly - you can verify this yourself using Redux Dev tools, then browsing the `wc/store/cart` store.\

I don't see any cart line items rendered in the DOM, though, so we know they're not just being hidden by CSS.

> how can I facilitate a verbose logging from these blocks in order to analyze what's happening here?

Currently we don't have anything like this that'd be useful for you in this case. What I recommend doing is disabling plugins one by one until it works again, and that way we'll find the culprit.

I see cwicly is adding its own data to the Store API Cart Item responses, so _maybe_ that has something to do with it.

If you could try that and let us know if it solves it, or if the issue is still present after disabling all plugins that would be really helpful.

Thanks!

**Reply by batonac** [reply]
Thanks @opr. I narrowed the issue down to the Cwicly theme/plugin, so I'll reach out to their team.

**Reply by opr** [reply]
That's good to hear, do let us know if you need any more help! I'll close this discussion now since it looks like it's caused by a third party, but feel free to reopen it if you think this is actually something we need to address, or open an issue if there's a bug we can fix.

Thanks!
